### PR TITLE
Web Vitals extension v1 blog post

### PR DESCRIPTION
**Pull request by rviscomi**
Draft of the blog post announcing v1 of the Web Vitals extension and demoing the new field data feature.

TODO:

- [x] add subhead/description metadata
- [x] update `date` metadata when ready to publish
- [x] fix broken video embed

cc @addyosmani

### CODE REVIEW

**Comment by mihajlija** [comment]
@rviscomi What's the deadline for publishing this? In future, please first submit a [content proposal form](https://docs.google.com/forms/d/e/1FAIpQLSc65CDClpUu7R2ECacLz3B1a6hOCWdFAk2vkWXIbZjzNSXq_Q/viewform?resourcekey=0-t3rJNkt5V2-iE2N42KhrhQ) (which asks for details such as the deadline). This helps the tech writing team plan and prioritize reviewing tasks.

**Comment by rviscomi** [comment]
@mihajlija it's blocked on v1 of the extension getting approved in the Chrome Web Store, which may take about a week. @addyosmani has more context on it.

Sorry for not submitting a content proposal, I mistakenly assumed that didn't apply to blog posts.

**Comment by mihajlija** [comment]
@rviscomi Noted, thanks!

**Comment by rviscomi** [comment]
@addyosmani it looks like the extension has been updated to v1.0.1 in the [store](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en). Are we good to publish this announcement from your perspective? If so we can open the PR up for review and aim to publish sometime this week, if that also works for @mihajlija.

**Comment by mihajlija** [comment]
If you're not expecting any big changes to the text, I can start review and preemptively approve so you can merge whenever ready. 

@rviscomi Please rewrap the file to break at 100 chars, linter doesn't enforce it but it will make the review easier.

**Review by addyosmani — APPROVED** [review]
LGTM. The latest version of the extension is live and I'm happy for this post to land whenever Rick is ready.

**Comment by addyosmani** [comment]
> If you're not expecting any big changes to the text, I can start review and preemptively approve so you can merge whenever ready. 
> 
> 
> 
> @rviscomi Please rewrap the file to break at 100 chars, linter doesn't enforce it but it will make the review easier.

I believe the post should be good to review :)

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:2]: I'd suggest a more straightforward title, rather than a clever one. You want people to know exactly what's the post about before clicking it. How about something like "Field data comes to the Web Vitals extension"?

Comment (rviscomi) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:2]: I actually like the clever title. It alludes to the value of field data and it helps tie together a metaphor used in the post with hero image.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:29]:
```suggestion
[Core Web Vitals](/vitals) are the most important metrics to measure to
understand your users' experiences, because when users have good experiences,
[good things happen](https://wpostats.com/)! The [Web Vitals extension](https://chrome.google.com/webstore/detail/web-vitals/ahfhijdlegdabablpippeagghigmibma?hl=en)
for Chrome is one tool in the [Web Vitals toolbox](/vitals-tools/) that shows you 
performance data about the pages you visit as you browse the web. 

Page performance depends on many factors, including hardware and network quality. How
you experience a page on your machine may be very different from how majority of users
experience it. That's why the version 1.0 of the Web Vitals extension includes a new feature
that integrates real-user data from the [Chrome UX Report](/chrome-ux-report/) (CrUX) 
with your local Core Web Vitals measurements. This feature displays your local measurements
in the context of how other users have experienced the same pages. It comes with
a sparkly new UI and I'm excited to show you how it works.
```

Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:29]: - Chunking paragraphs for readability
- Replaced some instances of _experience_ (nebulous) with _measurement_ (concrete)
- Rewrote the rhetorical and "me/thee" part to be simpler and straightforward

Please edit the suggestions if I changed the meaning somewhere, but maintain the simpler style.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:41]: I suggest adding a subheading here, and for every other distinct section. This could be as simple as "CrUX and field data", but it will make the post easier to scan, and easier for readers to go back to specific sections.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:29]: I've also trimmed "pages and websites" to pages. I know that's simplistic, if you think it's important to be precise in the intro just edit it back in.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:51]:
```suggestion
CrUX is a public dataset of real-user experiences in Chrome. It powers some of the critical tools
in the Core Web Vitals workflow like [Search Console](https://support.google.com/webmasters/answer/9205520#about_data)
and [PageSpeed Insights](https://developers.google.com/speed/docs/insights/v5/about#crux).
The raw data for millions of websites is also publicly queryable in the [CrUX dataset on BigQuery](/chrome-ux-report-bigquery)
and the [CrUX API](/chrome-ux-report-api). For this update to the Web Vitals
extension, we've integrated it with the page and origin-level desktop data in the CrUX API. User
experience data is broken down into three qualitative ratings: good, needs improvement, and poor.
The thresholds used for each rating are documented in the guides for each of the Core Web Vitals
metrics: [LCP](/lcp/#what-is-lcp), [FID](/fid/#what-is-fid), and
[CLS](/cls/#what-is-a-good-cls-score). So for example, the CrUX API can tell us that
```

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:54]: Breakpoint for a subheading. "New way of displaying CWV data", "How the extension displays data", "New extension UI", something along those lines.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:66]:
```suggestion
In the new Web Vitals extension interface, your local experience is layered on
top of the broader distribution of real-user experiences. It's like when you're in an expansive
```

Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:66]: Again, I'd replace "experience" with "data" or "measurements" here if that works. Not sure about technical correctness, so I'll leave it to you to decide.

Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:66]: Ah, looking closely at the extension UI, I see it uses this wording. I won't edit it anymore, though I still suggest keeping the edit in the intro. Intro is the hook, better to be simple even if it's not quite technically correct.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:73]:
```suggestion
"YOU ARE HERE" to help you orient yourself and find where you're going.

The distributions of real-user data from CrUX are laid out on a horizontal bar for each metric. 
Your local experiences are displayed as single points on those horizontal bars. The position of
your local experience on the horizontal bar indicates where that experience falls relative to the
real-user distributions. This should make it immediately apparent when your local
experience is very different from other users'. For example, if you have a slow LCP experience and
only 1% of users have similar experiences, something unusual must have happened.
```

Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:73]: Paragraph break and shorter simpler sentences.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:84]: Good place for a subheading. Something like "Displaying page-level field data when origin-level data is not available"

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:93]:
```suggestion
Some pages don't have URL-level data in the CrUX dataset. This can happen
because the page is so new that it hasn't been picked up yet or because the page has insufficient
traffic to be included in the dataset. In cases like these, Web Vitals extension falls back to more
granular origin-level data whenever possible. This data represents the aggregate user experiences
on all pages of the website, so while it may not be immediately relevant to your particular page
experience, it should still offer some insight into how users experience the site as a whole.

Another common case is to see a "Waiting for input.." message. This is because FID is the only
Core Web Vital that requires user interaction to be measured, so the extension won't have any local FID data
to show until you interact with the page.
```

Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:93]: The UI shows "Waiting for input..." message in my version so I've edited that.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:117]:
```suggestion
There are some less common edge cases that you may run into while using the extension. 

It can happen that entire origins have insufficient data. In that case, the extension will just show you
how your local experiences performed. 

How you interact with the page may also affect the relevance of the data. For example, loading the
page in a background tab will count against the LCP metric the entire time until you foreground the page. 
If it takes you 90 seconds to switch to the tab, the LCP might appear as 91.5 seconds. When this happens, a little
warning icon will appear next to your LCP value to warn you of the artificial inflation. Note that
these kinds of irrelevant LCP values are ignored in the CrUX dataset.

One last edge case to be aware of is specific to FID. Because it depends on user interactions, that makes it
less likely to be available in the dataset for pages and websites with fewer page views. So if you're
visiting a page like that, you may only see field data for LCP and CLS. It's worth noting for the
sake of completeness that the same could happen for LCP or CLS in extremely rare situations, but
this issue is predominantly affected by FID.
```

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:94]: Add a subheading, maybe "Edge cases".

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:58]:
```suggestion
{% Img src="image/STd8eW8CSiNp5B1bX0R6Dww2eH32/EaVNEuQ6gxVLtHYNkSZj.png", alt="Screenshot of the
Web Vitals extension showing an explanation of how the local LCP experience relates to real-user
desktop data from the field.", width="800", height="549", class="screenshot" %}
```

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:127]: Subheading cue, could be "Comparing field data from mobile devices" or similar

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:128]: Also might be a good idea to mention the desktop thing somewhere earlier in the post.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:137]:
```suggestion
experiences from the field. After all, this extension is only available on desktop versions of
Chrome, so it'd be most relevant to see how users under similar conditions experience the
page or origin.

It's important to understand phone users' experiences too, so the extension has an advanced setting on
the Options page that lets you see how your local experience compares to phone data from the field.

To enable phone data in the extension, follow these steps: 

1. Right click on the extension icon in the toolbar and select **Options**. 
2. Check the **Compare local experiences to phone field data** option. 

The UI will update in a few places to indicate which mode you're in. Be aware that real phone
users' experiences can be _very different_ from that of desktop users, so use this feature with discretion.
```

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:138]: Subhead cue, "Feedback welcome" or "Try out the Web Vitals extension"

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [inline src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:146]:
```suggestion
(feature requests, bug reports, anything), let us know in the
[open-source repository](https://github.com/GoogleChrome/web-vitals-extension) on GitHub.
I hope it helps you better understand where your local experiences are in relation to other
real users from the field!
```

Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:146]: The tone is polite enough already :)

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:2]: Here's my reasoning: consider all the different ways people can reach a link to this post without the context of the hero image such as newsletters, hacker news style aggregators, etc. The image can also fail to load, and then there are people with poor vision. The title is cute and clever, but it obfuscates what the post is about. I wasn't sure until the second paragraph. Not saying you can't have fun, but in this case alternatives would be more effective.

If you still want to keep it, I think "Find your way with field data in the Web Vitals extension update" is better than "new field data".

**Review by mihajlija — CHANGES_REQUESTED** [review]
One more request: please add the `screenshot` class to all images.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:145]: No need to edit, I missed it. I'm gonna break up that paragraph too so it's easier to parse.

File: src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md
Comment (mihajlija) [reply src/site/content/en/blog/field-data-in-the-web-vitals-extension/index.md:43]: You placed them right, I probably lost my place in the file. Headings have to be sequential though, so H2. I'll change them.

**Review by mihajlija — APPROVED** [review]

**Comment by mihajlija** [comment]
Good to go, merge when ready! 🚀

**Comment by rviscomi** [comment]
Thanks for the thorough review @mihajlija! The post is in a much better state now.

**Comment by rviscomi** [comment]
The post is live: https://web.dev/field-data-in-the-web-vitals-extension/
Announcement tweet: https://twitter.com/rick_viscomi/status/1410979927684419592

Thanks again everyone!

**Pull request merged**
